src/java/main/org/apache/zookeeper/server/persistence/Util.java

```diff
@@ -83,7 +86,7 @@ public static URI makeFileLoggerURL(File dataDir, File dataLogDir,String convPol
      * @return file name
      */
     public static String makeLogName(long zxid) {
-        return "log." + Long.toHexString(zxid);
+        return LOG_FILE_PREFIX + "." + Long.toHexString(zxid);
```

Comment:
I wonder if LOG_FILE_PREFIX included the dot, would be better for the validation.

Reply:
I deliberately excluded the dot from LOG_FILE_PREFIX and SNAP_FILE_PREFIX after I checked that in FileTxnLog and FileSnap classes the prefix argument (like "log", "snapshot") in method calls is passed without the dot. I wanted to avoid confusion.

Reply:
Okay.